### DISCUSSION TITLE
Issue installing python tools after version 32.74.3

### DISCUSSION THREAD

**Discussion by dcrowe** [renovatebot/renovate#16746] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.127.0

### Please select which platform you are using if self-hosting.

Bitbucket Cloud (bitbucket.org)

### If you're self-hosting Renovate, tell us what version of the platform you run.

N/A

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Hi, 
We self-host renovate-bot using Docker, but we install additional versions of python because some of our repos are using older versions of python.

Our Docker file looks like:
```
FROM renovate/renovate
USER root

# many of our projects want python 3.9
ARG PYTHON39_VERSION=3.9.6
RUN install-tool python $PYTHON39_VERSION && \
    /usr/local/python/$PYTHON39_VERSION/bin/python -m pip install pipenv

ENV LC_ALL=C.UTF-8
ENV LANG=C.UTF-8
```

This has worked well for a couple of years. Thanks!
But it has stopped working after renovate version 32.74.3.
It now fails with an error that appears to be due to the `PYTHONHOME` env var being set. This issue doesn't appear to be present in the source `containerbase/buildpack` docker image.

### Relevant debug logs

<details><summary>Logs</summary>

```
 > [2/2] RUN install-tool python 3.9.6 &&     /usr/local/python/3.9.6/bin/python -m pip install pipenv:                                                              
#5 0.546 installing v2 tool python v3.9.6                                                                                                                            
#5 24.51 Using prebuild python                                                                                                                                       
#5 29.25 Requirement already satisfied: pip in /opt/buildpack/tools/python/3.9.6/lib/python3.9/site-packages (21.1.3)                                                
#5 29.72 Collecting pip                                                                                                                                              
#5 29.96   Downloading pip-22.2-py3-none-any.whl (2.0 MB)
#5 31.51 Installing collected packages: pip
#5 31.51   Attempting uninstall: pip
#5 31.51     Found existing installation: pip 21.1.3
#5 31.65     Uninstalling pip-21.1.3:
#5 31.66       Successfully uninstalled pip-21.1.3
#5 32.99 Successfully installed pip-22.2
#5 32.99 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#5 33.87 Files removed: 2
#5 33.94 linking tool python v3.9.6
#5 33.99 Python 3.9.6
#5 34.25 pip 22.2 from /opt/buildpack/tools/python/3.9.6/lib/python3.9/site-packages/pip (python 3.9)
#5 34.31 /bin/bash: /usr/local/python/3.9.6/bin/python: No such file or directory
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by rarkins** [comment, marked as answer]
Please raise in the containerbase/buildpack repo instead

**Reply by dcrowe** [reply]
Have raised the issue here: https://github.com/containerbase/buildpack/issues/461

**Reply by dcrowe** [reply]
The summary is that since buildpack version 4.1.0 it has not been possible to run multiple versions of python in parallel (which is needed to support multiple projects that target different python revisions)

There is a workaround though:
```
RUN rm /opt/buildpack/env.d/python.sh
ENV PATH="${PATH}:/usr/local/python/${PYTHON38_VERSION}/bin"
ENV PATH="${PATH}:/usr/local/python/${PYTHON39_VERSION}/bin"
```

**Comment by viceice** [comment]
python is now located at `/opt/buildpack/tools/python`.

This is not a bug, as we never garanteed that your usecase should work

**Comment by rarkins** [comment]
Converted to discussion
